src/bakery_manager_mobile/lib/pages/task_details_page.dart

```diff
@@ -367,7 +402,7 @@ class _TaskDetailPageState extends State<TaskDetailPage> {
                         if (confirmed) {
                           // Call API to delete task and handle success/failure
                           try {
-                            // await ApiService.deleteTask(task.taskID ?? '');
+                            await ApiService.deleteTask(task.taskID);
```

Comment:
idk if you were still planning on it but Delete Task doesn't work

Reply:
Correct it needs a backend update that hopefully gets done